client/gameObjects/Skeleton.js

```diff
@@ -284,6 +317,15 @@ export default class Skeleton extends Phaser.GameObjects.Image {
       }
     }
 
+    // Handle other players only
+    if (!this.isMainPlayer) {
```

Comment:
else in line 318 should do the job?

Reply:
I have to change it to `} else if (!this.isMainPlayer) {` as ESLint actually forces joining `else` with `if` and ends up with `else (this.isDead) {` which is too chaotic for me.